app/services/proofing/aamva/applicant.rb

```diff
@@ -1,11 +1,30 @@
 # frozen_string_literal: true
 
 require 'date'
-require 'hashie/mash'
 
 module Proofing
   module Aamva
-    class Applicant < Hashie::Mash
+    Applicant = Struct.new(
```

Comment:
Another option is to use `Data.define` for read-only versions of these objects. However, the specs seemed to rely on hash key access, which structs provide, but data objects do not

Reply:
Also open to using `RedactedStruct` if we think there's risk of these being copy-pasted in the console too

Reply:
I vote for RedactedStruct, on general paranioa principles, but I don't feel strongly about it.

Reply:
done in 9280eb4f2a983702893f6b57382701601314b9a5